content/consistency-standardization.md

```diff
@@ -0,0 +1,66 @@
+Title: WHY are we pushing consistency and standardization?
+Date: 2022-03-25
+Category: Posts
+Tags: eliminate-waste, technical-excellence
+Slug: consistency-standardization
+Author: Willy-Peter Schaub
+Summary: Balancing autonomy with consistency and standardization.
+
+**WHY** do we keep mentioning **consistency** and **standardization** like a broken record in [Journal 1 - What is origin of our common engineering team?](/common-engineering-journal-1.html) and many other posts? 
+
+Before we explore the answer, let us explore our understanding of consistency and standardization.
+
+---
+
+# Consistency
+
+**Consistency** is coherence, a key pillar in engineering. It empowers a unified whole, such as a common engineering system or a complex solution that is maintainable and extensible.
+
+![Consistency](../images/consistency-standardization-1.png) 
+
+The previous illustration shows twelve **consistent** squares that have the same height, width, and colouring. A great example of consistency is [LEGO](https://www.lego.com/) which offers us an assortment of consistent building blocks, with which we can follow directions or experiment in creating simple or complex objects, based on consistent building blocks.
+
+--- 
+
+# Standardization
+
+**Standardization** is another pillar of engineering. It is the process of observing guidance, guardrail, standard, or governance.
+
+![Standardization](../images/consistency-standardization-2.png) 
+
+In the illustration, above, we have defined a guardrail that the height and width of the rectangle must be equal and that all lines are straight and parallel - a recipe to create a rectangle, also known as a square.
+
+Standardization is pivotal with designs, producing **consistent** interfaces, solutions, and uniformity that empowers people to be more effective.
+
+---
+
+# Back to the question ... why do we promote consistency and standardization?
+
+Why we love consistency and standardization:
+
+- **Consistency improves adoption** - Once we become familiar with a process or product, resistance to explore and embrace others will evaporate. For example, if you know how to use Office 365 Word, you can navigate yourself around Office 365 Excel or PowerPoint with familiarity and without friction.
+
+- **Consistency reduces cost and time** - Usability and learnability improves with consistency, which empowers engineering to deliver value quicker and continuously. Reducing time and anomalies, reduces cost - a favourable side-effect for investors and researchers.
+
+- **Consistency empowers automation** - We frequently say that "_if you do something twice or trice, consider automation!_" While this is solid guidance, automation becomes a lot easier if you have a consistent process and products at your disposal.
```

Comment:
Sorry, being nit-picky here: "Thrice" is misspelled.

Reply:
I love nit-picky and pedantic reviews :) Fixed.